src/transformers/generation_flax_utils.py

```diff
@@ -209,8 +211,10 @@ def generate(
 
             input_ids (`jnp.ndarray` of shape `(batch_size, sequence_length)`):
                 The sequence used as a prompt for the generation.
-            max_length (`int`, *optional*, defaults to 20):
-                The maximum length of the sequence to be generated.
+            max_length (`int`, *optional*, defaults to `model.config.max_length`):
+                **DEPRECATED** The maximum length of the sequence to be generated. Prefer the use of `max_new_tokens`.
```

Comment:
```suggestion
                **DEPRECATED** The maximum length of the sequence to be generated. Prefer the use of `max_new_tokens` which ignore the number of tokens in the prompt.
```
(Here and in all other docstrings.)